src/types.ts

```diff
@@ -1,8 +1,8 @@
-
+// Operations that can be performed on the GPU.
```

Comment:
Prefer a JSDoc comment here.

Reply:
Additionally, consider describing what a "Channel" is. I still don't fully understand what they are and what they do after reading this file.

Reply:
FYI: 
```ts
/**
 * A channel represents the information necessary to map a single dimension
 * (x, y, color, jitter, etc.) from dataspace to a visual encoding. It is used
 * to construct a scale and to pass any other necessary information to assist
 * in converting tabular data to visual representation. In some cases it is a scale;
 * in others it is parameters used to define a function on the GPU.
 * 
 * The names and design are taken largely from channels as defined in Vega-Lite, 
 * but the syntax differs for a number of operations.
 * https://vega.github.io/vega-lite/docs/encoding.html
 */
```